### PR TITLE
refactor(README): change README - issue#106

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
Signed-off-by: Parikshit Hooda <phooda804@live.com>

# Issue #<NUMBER HERE>
change readme to fix typo

### Changes
- optionall -> optional

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #106 
- Pull Request #<NUMBER HERE>

### CODE REVIEW

**Comment by Parikshit-Hooda** [comment]
@sstone1 @mbwhite @dselman  Guys, please merge this as part of hacktoberfest.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Concerto is a lightweight 100% JavaScript schema language and runtime. It works
 
 Things you can do using Concerto:
 - Define an object-oriented model using a domain-specific language that is much easier to read and write than JSON/XML Schema, XMI or equivalents. The metamodel gives you "just enough" expressivity to capture real-world business models, while remaining easy to map to most runtime environments.
-- Optionall edit your models using a powerful [VS Code add-on](https://marketplace.visualstudio.com/items?itemName=accordproject.cicero-vscode-extension) with syntax highlighting and validation
+- Optional edit your models using a powerful [VS Code add-on](https://marketplace.visualstudio.com/items?itemName=accordproject.cicero-vscode-extension) with syntax highlighting and validation
```
Comment (mttrbrts) [inline README.md]: This should say `Optionally`

Comment (Parikshit-Hooda) [reply README.md]: okay. let me fix this.

**Comment by jeromesimeon** [comment]
In #114

**Pull request closed without merging**